src/domain/CheckNotificationStatusRecord.ts

```diff
@@ -49,7 +49,10 @@ export const makeCheckNotificationStatusRecord =
                 t.exact(NewNotificationRequestStatusResponse).encode({ ...n, notificationRequestStatus: 'ACCEPTED' })
             )
           ),
-          O.map((response) => ({ statusCode: 200 as const, returned: response })),
+          O.map((response) => ({
+            statusCode: 200 as const,
+            returned: { ...response, retryAfter: env.retryAfterMs / 1000 },
```

Comment:
```suggestion
            returned: { ...response, retryAfter: env.retryAfterMs },
```
Not sure we should divide by 1000. According to the doc:
> retry-after: Numero di millisecondi di attesa prima di effettuare una nuova lettura di eventi.

so I guess it should be `env.retryAfterMs`.

Reply:
There are two different *retry after* field, one in second and the other one in millisecond. This [one](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/pagopa/pn-delivery/develop/docs/openapi/api-external-b2b-pa-v1.yaml#/SenderReadB2B/retrieveNotificationRequestStatus) is in second and this [one](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/pagopa/pn-delivery-push/develop/docs/openapi/api-external-b2b-webhook-v1.yaml#/Events/consumeEventStream) in millisecond

Reply:
That's right! My bad!